### PR TITLE
add:codrops in animations section

### LINKED ISSUE #1429: [Add] codrops in animation section
### Site URL

https://tympanus.net/codrops/

### Description of the above provided site

A web design and development blog with tutorials and examples of creative animations and interactions.

### Category

Frontend

### Sub Category

animations

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Akanksha-202**
## Fixes Issue

Closes #1429 

## Changes proposed

- Added codrops website to the animations section

## Screenshots

image

### CODE REVIEW

File: database/frontend/animations.json [hunk 1]
```diff
@@ -117,5 +117,12 @@
     "url": "https://animate.style/",
     "category": "frontend",
     "subcategory": "animations"
+  },
+  {
+    "name": "codrops",
+    "description": "A web design and development blog with tutorials and examples of creative animations and interactions.",
```
Comment (Anmol-Baranwal) [inline database/frontend/animations.json:123]:
```suggestion
    "description": "Codrops is a leading web design and development blog that offers tutorials, examples, and inspiration for creative animations and interactions.",
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
What do you think?

Suggested description looks concise right. Let me know if you think otherwise.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
looks good to me

**Pull request merged**